python/test/unit/language/test_core.py

```diff
@@ -163,6 +163,15 @@ def check_type_supported(dtype, device):
         ]:
             pytest.skip("bfloat16 and float8 are not supported in the interpreter")
 
+class MfmaLayout:
+    def __init__(self, version, warps_per_cta, instr_shape, is_transposed):
+        self.version = version
+        self.warps_per_cta = warps_per_cta
+        self.instr_shape = instr_shape
+        self.is_transposed = is_transposed
+
+    def __str__(self):
+        return f"#{GPU_DIALECT}.amd_mfma<{{versionMajor={self.version[0]}, versionMinor={self.version[1]}, warpsPerCTA = {str(self.warps_per_cta)}, instrShape={str(self.instr_shape)}, isTransposed = {str(self.is_transposed).lower()}}}>"
```

Comment:
nutpick! I believe `str` conversion is not needed for some  cases
```suggestion
        return f"#{GPU_DIALECT}.amd_mfma<{{versionMajor={self.version[0]}, versionMinor={self.version[1]}, warpsPerCTA = {self.warps_per_cta}, instrShape={self.instr_shape}, isTransposed = {str(self.is_transposed).lower()}}}>"
```

Reply:
Done.